admin/pages/social.php

```diff
@@ -83,7 +84,10 @@
 		</p>
 
 		<p class="desc">
-			<?php _e( 'Add Twitter card meta data to your site\'s <code>&lt;head&gt;</code> section.', 'wordpress-seo' ); ?>
+			<?php
```

Comment:
You left `class="desc"` here while you removed it in other places. Just checking if that was intentional.